sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -197,8 +197,8 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
   }
 
   /**
-   * Buckets the output by the given columns. If specified, the output is laid out on the file
-   * system similar to Hive's bucketing scheme.
+   * Buckets the output by the given columns. Note that the output follows a Spark SQL specific
+   * bucketing scheme based on the Hive scheme.
```

Comment:
minor though, WDYT this update? @cloud-fan @HyukjinKwon

Reply:
I feel the original one is clear enough as it says "similar to". Maybe we can add one more sentence: `..., but with a different bucket hash function and is not compatible with Hive's bucketing.`

Reply:
ok

Reply:
updated.